vivo-project/VIVO
# [VIVO-1656] Extension mechansim for passing properties to templates

**[VIVO-1656](https://jira.duraspace.org/browse/VIVO-1656)**:

# What does this pull request do?
Uses the extension mechanism to pass the Plum Analytics and AltMetric options to the individual templates.

Requires the corresponding pull request from Vitro.

# What's new?
No change in functionality, this is an architectural improvement.

# How should this be tested?
Visit the page of any "individual" in VIVO. Should work exactly as it did before.

In particular, check publications that previously display Plum Analytics / AltMetric badges - these badges should still appear as before.

# Interested parties
@VIVO-project/vivo-committers
